frontend/src/common/ImageComponent.js

```diff
@@ -0,0 +1,22 @@
+import React, {useEffect, useState} from 'react'
+
+function ImageComponent({imageId}) {
+
+  const [state, setState] = useState({
+    base64String: null
+  })
+
+  useEffect(() => {
+    fetch('/api/image/' + imageId, {method:"GET"})
+      .then((response) => response.json())
+      .then((json) => setState({base64String: json.base64String}))
+  }, [imageId])
+
+  return (
+    <>
+      {state.base64String && <img src={state.base64String} alt="image" style={{width: '100%'}}/>}
```

Comment:
Maybe we can have a placeholder image for not found base64s

Reply:
An art item or an event can not be created without an image in the current configuration of the endpoints. But I still added a placeholder GIF to act as a loading indicator:

![image](https://user-images.githubusercontent.com/57228345/204098333-b4bf4b72-c421-40cd-a0a0-6b47ea01921a.png)